css/main.css

```diff
@@ -25,6 +26,10 @@ hr {
     padding: 0;
 }
 
+a, a:hover {
+  text-decoration: none;
+}
+
```

Comment:
I don't think you need the `:hover` part of this — links don't have a hover style by default.